<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign</class>
 <widget class="QWidget" name="sign">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>507</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
                font-size:24px;
                font-family:sans-serif;
                }
                Form{
                background-color:white;
                }
                QFrame{
                background:rgba(0,0,0,0.8);
                border-radius:15px;
                }
                QPushButton{
                background:#03a9f4;
                color:#fff;
                border-radius:15px;
                }
                QLineEdit{
                border-radius:15px;
                color:#03a9f4;
                }
                QLabel{
                color:#fff;
                background:transparent;
                font-size:30px;
                }
            </string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px;
border-radius:5px;
padding:2px 4px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>注册/注销</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="signButton">
       <property name="styleSheet">
        <string notr="true">border:2px white;
border-radius:5px;
padding:2px 4px;
color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
padding:2px 4px;
border:2px white;
color:0x000000;</string>
       </property>
       <property name="text">
        <string>注销账户</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="signID">
       <property name="styleSheet">
        <string notr="true">border:2px;
border-radius:5px;
padding:2px 4px;
color:gray</string>
       </property>
       <property name="placeholderText">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="signPswd">
       <property name="styleSheet">
        <string notr="true">border:2px;
border-radius:5px;
padding:2px 4px;
color:gray</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
